playground/backend/internal/code_processing/code_processing_test.go

```diff
@@ -628,8 +628,8 @@ func Test_getRunOrTestCmd(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := getExecuteCmd(tt.args.isUnitTest, tt.args.executor, tt.args.ctxWithTimeout); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("getExecuteCmd() = %v, want %v", got, tt.want)
+			if got := getExecuteCmd(tt.args.isUnitTest, tt.args.executor, tt.args.ctxWithTimeout); got.String() != tt.want.String() {
```

Comment:
Could you try this? https://github.com/google/go-cmp
The problem with comparing strings is that the test becomes difficult to identify the exact reason for why it failed without drilling down further.